examples/weaviate/requirements.txt

```
@@ -0,0 +1,4 @@
+jupyter
+tqdm
+weaviate-client
+unstructured
```

Actually I think we need this?
```suggestion
unstructured[local-inference]
```